code/modules/surgery/organs/autosurgeon.dm

```diff
@@ -103,9 +103,55 @@
 	uses = 1
 	starting_organ = /obj/item/organ/cyberimp/arm/syndie_mantis
 
-/obj/item/autosurgeon/organ/syndicate/syndie_mantis/l
+/obj/item/autosurgeon/organ/syndicate/syndie_mantis/AltClick(mob/living/user) //Basically this combines screwdriver with an insert_organ proc because I can't be assed
+	var/obj/item/I = storedorgan
+	var/atom/drop_loc = user.drop_location()
+	if(!storedorgan)
+		return
+	for(var/atom/movable/AM in src)
+		AM.forceMove(drop_loc)
+	if(!istype(I, /obj/item/organ/cyberimp/arm/syndie_mantis/l))
+		for(var/obj/item/organ/cyberimp/arm/syndie_mantis/R in drop_loc)
+			R.Destroy()
+		var/obj/item/organ/O = new /obj/item/organ/cyberimp/arm/syndie_mantis/l(src)
+		insert_organ(O)
```

Comment:
so, why do you need to recreate the implant here to a type with just a different zone? why not just change the zone manually and then call SetSlotFromZone(), seems way better